docs/extensions/extension_quickstart.md

```diff
@@ -43,7 +39,7 @@ phylum extension install ./my-extension
 Once successfully installed, our extension can be executed by using its name as
 a subcommand for the phylum CLI:
 
-```console
+```sh
```

Comment:
This is wrong. No line in this codeblock is valid `sh`.

Reply:
`console` is also wrong because it is not a [supported language for Docusaurus code blocks](https://docusaurus.io/docs/markdown-features/code-blocks#supported-languages). Basically, the choice of code block language needs to come from [this list of supported languages](https://prismjs.com/#supported-languages). Which one from there is preferred?

Reply:
> `console` is also wrong because it is not a [supported language for Docusaurus code blocks](https://docusaurus.io/docs/markdown-features/code-blocks#supported-languages).

Does docusaurus enforce this? Because that's not really how Markdown works...

Here is the relevant excerpt from [the CommonMark spec][1]:

[1]: https://spec.commonmark.org/0.30/#code-fence

> An [info string](https://spec.commonmark.org/0.30/#info-string) can be provided after the opening code fence. Although this spec doesn’t mandate any particular treatment of the info string, the first word is typically used to specify the language of the code block. In HTML output, the language is normally indicated by adding a class to the code element consisting of language- followed by the language name.

So when you say this is "wrong"... Do you just mean that no syntax highlighting is applied? That's not really a problem.

Reply:
GitHub uses Linguist to perform language detection and syntax highlighting. `console` [appears to be an alias for `shellsession`](https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml#L6476-L6487) (and `bash session`).

Docusaurus uses PrismJS to perform code block syntax highlighting. `console` is not [one of the supported languages](https://prismjs.com/#supported-languages) it detects, but `shellsession` and `shell` (with an `sh` alias) both are...making either of them a common choice.

Docusaurus comes with a subset of [commonly used languages](https://github.com/FormidableLabs/prism-react-renderer/blob/v1.3.0/src/vendor/prism/includeLangs.js). Bash is included in that list (with accepted aliases of `bash`, `sh`, and `shell`)...making the change here a natural substitution for `console`.

If that is still not desired, then additional languages, taken from the list of PrismJS languages, can be used if they are added to the `docusaurus.config.js` file, in the `themeConfig.prism.additionalLanguages` list. This PR still needs to add that setting with the additional code block languages in use...currently only `toml` but maybe an additional entry for whatever is decided here to use for representing a shell/console command/session.

Once it does, "enforcement" happens when trying to build the site since incorrect entries in that list will generate a build error. So...yes...currently these code fence "info strings" are ignored when they don't match one of the configured languages and no syntax highlighting is the result.

Reply:
> Docusaurus comes with a subset of [commonly used languages](https://github.com/FormidableLabs/prism-react-renderer/blob/v1.3.0/src/vendor/prism/includeLangs.js). Bash is included in that list (with accepted aliases of `bash`, `sh`, and `shell`)...making the change here a natural substitution for `console`.

You're confusing two very different "languages"...

1. Shell scripts (i.e., `sh`, `shell`, `bash`, etc.)
2. Console output (i.e., `console`, `shellsession`, `shell-session`, etc.)

We are in the second group here and we need to stay in that group. Annoyingly, PrismJS has [rejected `console`](https://github.com/PrismJS/prism/pull/2209) as an alias for `shell-session`. But it seems like `shellsession` is a widely supported value.

```suggestion
```shellsession
```